src/Orleans.Core/Serialization/SerializationManager.cs

```diff
@@ -173,6 +166,15 @@ internal IRuntimeClient RuntimeClient
             RegisterSerializationProviders(serializatonProviderOptionsValue.SerializationProviders);
         }
 
+        public void SetApplicationPartManager(ApplicationPartManager applicationPartManager)
```

Comment:
would `RegisterSerializers` be a more appropriate name? this is not "setting the app manager for later use", it's commanding the SM to register the serializers (from the app manager) *now*